### PR TITLE
Decrease terraformer log level

### PR DESCRIPTION
**Pull request by timebertt**
Decrease terraformer log level

### CODE REVIEW

**Review by rfranzke — APPROVED** [review]
/lgtm

**Review by ialidzhikov — APPROVED** [review]
/lgtm

File: pkg/controller/infrastructure/actuator.go [hunk 1]
```diff
@@ -54,7 +54,6 @@ func newTerraformer(logger logr.Logger, restConfig *rest.Config, purpose string,
 	owner := metav1.NewControllerRef(infra, extensionsv1alpha1.SchemeGroupVersion.WithKind(extensionsv1alpha1.InfrastructureResource))
 	return tf.
 		UseV2(true).
```
Comment (ialidzhikov) [inline pkg/controller/infrastructure/actuator.go:56]: Side note out of context of this PR: we can also consider making v2 the default mode in the terraform pkg.

**Pull request merged**